drivers/boards/GENERIC_STM32_F103/board.mk

```diff
@@ -0,0 +1,5 @@
+# List of all the board related files.
+BOARDSRC = $(BOARD_PATH)/boards/GENERIC_STM32_F103/board.c
```

Comment:
this has to change to drivers if you want it to work iirc

Reply:
The Proton C setup has:
```makefile
BOARDSRC = $(BOARD_PATH)/boards/GENERIC_STM32_F303XC/board.c
```

So its set up similar, I think im having a bit of a *woosh* moment as im not seeing what is missing?